ods_ci/tests/Tests/0100__platform/0102__monitor_and_manage/0101__metrics/0103__check_alertmanager_config.robot

```diff
@@ -29,16 +28,33 @@ ${SECRET_NAME}          redhat-rhods-pagerduty
 *** Test Cases ***
 PagerDuty Dummy Secret Verification
      [Documentation]    Verification of PagerDuty Secret
-     [Tags]  Sanity
-     ...     Tier1
+     [Tags]  Smoke
      ...     ODS-737
+     ...     ODS-500
+     ...     RHOAIENG-13069
      ...     Deployment-Cli
-     ...       Monitoring
+     ...     Monitoring
      Skip If RHODS Is Self-Managed
      ${service_key}   Get PagerDuty Key From Alertmanager ConfigMap
      ${secret_key}    Get PagerDuty Key From Secrets
      Should Be Equal As Strings    ${service_key}   ${secret_key}   foo-bar
 
+Verify DeadManSnitch configuration
+    [Documentation]    Verification of DeadManSnitch configuration
+    [Tags]   Smoke
+    ...      ODS-648
+    ...      RHOAIENG-13268
+    ...      Monitoring
+    Skip If RHODS Is Self-Managed
+    ${rc}    Run And Return Rc
+    ...    oc get secret redhat-rhods-deadmanssnitch -n ${MONITORING_NAMESPACE}
+    Should Be Equal As Integers    ${rc}    0
+    ${secret_snitch_url}    Run
+    ...    oc get secret redhat-rhods-deadmanssnitch -n ${MONITORING_NAMESPACE} -o yaml | yq -r '.data.SNITCH_URL' | base64 -d
```

Comment:
I'm not sure how common the 'yp' command is on systems. While it is available, I don't think it is installed by default.